apps/agentic-chat/src/components/chat.tsx

```diff
@@ -21,7 +21,7 @@ export const Chat: React.FC = () => {
     status,
   } = useChat({
     api: 'http://localhost:8080/',
-    maxSteps: 5,
+    maxSteps: 10,
```

Comment:
Might as well

Reply:
Any rational behind this? I'm curious if at some point we would need even more, I guess it depends on the number of tools users right?

Reply:
We may or may not need even more in the future yeah, very much arbitrary!
Although 10 is already super generous for a single response I believe